### ISSUE TITLE
FR: Bind a different key to send control characters

### ISSUE THREAD

**Issue by charredUtensil** [lanoxx/tilda#505]
**tl;dr I would like to configure Tilda to send the ^C character when I press caps lock+C and use control+C for "copy"**

My home laptop is a Linux machine, on which I use Tilda, and for work, I use a Macbook. Apple's idea of using the meta-key for all the normal keyboard shortcuts and the control key for the control signals actually kind of makes sense - or at least it would if anybody else ever picked it up. For me, I just want a consistent set of keybindings for common functionality - and on top of that I have a keyboard that lacks a meta key. My solution so far was to swap the bindings on the Mac so the control key sends meta, caps lock key sends control, and alt and control both send alt. That at least gives me a consistent experience between MacOS and Linux for the normal shortcuts every GUI program has - ctrl+ A,S,C,V,X,Z. I also use iTerm on the Mac because it has the same pulldown functionality as Tilda. That leaves one key combination different between the platforms: on the Mac I now press caps lock+C to kill a program and control+C to copy, but within Tilda I have to press control+C to kill a program and control+shift+C to copy.

I would like my keybindings consistent across platforms for the sake of my muscle memory.
